tools/cli/commands/connect.py

```diff
@@ -52,6 +52,17 @@
     $ {0} {1} example-instance --zone us-central1-a""")
 
 
+wrong_user_message = (
+    'The specified Datalab instance was created for {0}, but you '
+    'are attempting to connect to it as {1}.'
+    '\n\n'
+    'Datalab instances are single-user environments, and trying '
+    'to share one will cause problems.'
+    '\n\n'
+    'If you are sure you want to do this, then re-run the command '
+    'with the `--no-user-checking` flag.')
```

Comment:
Suggest:
To override this behavior, re-run the command with the --no-user-check flag.